### PR TITLE
sys-devel/qconf: bumped to 2.5

### PR DESCRIPTION
**Pull request by Ri0n**
Signed-off-by: Sergey Ilinykh <rion4ik@gmail.com>
Closes: https://bugs.gentoo.org/682428

### CODE REVIEW

File: sys-devel/qconf/qconf-2.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils
+
+DESCRIPTION="./configure like generator for qmake-based projects"
+HOMEPAGE="https://github.com/psi-im/qconf"
+SRC_URI="http://psi-im.org/files/qconf/${P}.tar.xz"
+SRC_URI="https://github.com/psi-im/qconf/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```
Comment (a17r) [inline sys-devel/qconf/qconf-2.5.ebuild]: This is odd. There can only be one `SRC_URI`.

Comment (Ri0n) [reply sys-devel/qconf/qconf-2.5.ebuild]: Thanks. forgot this thing while tested. fixed now.

**Comment by Ri0n** [comment]
> couldn't help but notice the tarball is double the size now. Is that normal?

probably because it's gz as direct download from github. I can repack and upload to some other place. Maybe it will be safer wrt checksums. wdyt?

**Comment by a17r** [comment]
ah yes, I see. not strictly necessary, your call.

**Comment by Ri0n** [comment]
I can't find any complaints about changing checksums for github releases. So it's safe probably. Let's merge it as is.

**Comment by a17r** [comment]
(the only complaints you could probably get from packagers about why the change of packaging formats, we like predictability - but in Gentoo you are the packager :p)

**Pull request closed without merging**